Bug: IDE removing wire import line

So I've installed the plugin recently and wanted to test it out. It seems like when it is active, upon file save the IDE removes from import line `"github.com/google/wire"`, from `wire.go` file, and this in turn causes wire command to return error: 
```
wire: /home/dawidd/projects/di/wire.go:13:2: undeclared name: wire
wire: generate failed
```
The file looks as follows. It's the 9th line that disappears upon save: 
```
//go:build wireinject
// +build wireinject

package di

import (
	"gin_tutorial/data/repositories"
	"gin_tutorial/server/controllers"
	"github.com/google/wire"
	"github.com/sirupsen/logrus"
)

func InitializeTextController() controllers.TextController {
	wire.Build(controllers.NewTextController, logrus.New, repositories.NewTextRepository)
	return *new(controllers.TextController)
}

```
 This is not the case if the plugin is disabled or not installed. Let me know if I should provide you with any additional detail.

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -5,7 +5,7 @@ plugins {
 }
 
 group = "com.saumyaroy"
-version = "0.0.7"
+version = "0.0.8"
 
 repositories {
     mavenCentral()
@@ -13,7 +13,7 @@ repositories {
 
 intellij {
     type.set("GO")
-    version.set("2022.2")
+    version.set("2022.2.2")
 
     plugins.set(listOf("org.jetbrains.plugins.go"))
 }

--- a/src/main/resources/META-INF/plugin.xml
+++ b/src/main/resources/META-INF/plugin.xml
@@ -22,7 +22,7 @@
         implementationClass="com.saumyaroy.gowireutil.fileType.WireFileType"
         language="go"
         fieldName="INSTANCE"
-        fileNames="wire.go;wire_gen.go"
+        fileNames="wire_gen.go"
         />
 
     </extensions>

